static/css/accounts/edit_profile.css

```diff
@@ -0,0 +1,5 @@
+#edit_profile {
+    align-content: center;
+    margin-top: 5%;
+    margin-bottom: auto;
+}
```

Comment:
missing line

Reply:
In html  and css files the formating is without a new line I think